files/es/web/api/notification/dir/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Notification.dir
+slug: Web/API/notification/dir
+tags:
+  - API
+  - Notification
+  - Notifications
+  - Notifications API
+  - Property
+  - Reference
+  - dir
+translation_of: Web/API/Notification/dir
+browser-compat: api.Notification.dir
+---
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{SeeCompatTable()}}
```

Comment:
```suggestion
{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
```

Reply:
Hello, @cw118 Thanks for the useful feedback about linking issues. In this case, I tested the suggested change but it appears in English in the Spanish version, would you recommend to apply it anyway?

Reply:
Hmm, from what I can see in the English file, my suggested change is what [matches en-US](https://github.com/mdn/content/blob/main/files/en-us/web/api/notification/dir/index.md?plain=1#L14)

Reply:
Ah, I see what you mean. I initially misunderstood — I'll go look for where to add a translation for `{{securecontext_header}}` and share the file location so that you guys can translate it to Spanish :)

Reply:
Perfect, I will add `{{securecontext_header}}` to this PR and I'll start to translate [secureContextGeneric.ejs at mdn/yari](https://github.com/mdn/yari/blob/main/kumascript/macros/secureContextGeneric.ejs).